### ISSUE TITLE
Creating Landing Page

### ISSUE THREAD

**Issue by shrishti-c** [Ayush-projects/chords#17]
Hey... I'm GSSOC21 participant. Since, it's being built from scratch, I would love to create landing page for Chords. Please assign me this.

**Comment by Chaitalishetty** [comment]
Hi I want to contribute to this. I'm a GSSOC21 participant. I can do the front-end part.

**Comment by shairanshrawat** [comment]
I am a participant in GSSOC 2021. I want to contribute to this issue and work on the same. Please could you assign me this issue as a part of GSSOC 2021?
Thank you

**Comment by Priyanka7011** [comment]
Hi ! I am a GSSOC 2021 participant.
I am a UI designer .I can create a user friendly and attractive landing page for the website.
I would love to contribute to this issue and will give my best.
Please assign this to me.
Thank You!!

**Comment by Ayush-projects** [comment]
Yes, sure. You can do it after we push the frontend base code to the repo.

**Comment by shrishti-c** [comment]
Sure Sir. Will we be notified once its get pushed so that we can start working?

**Comment by shairanshrawat** [comment]
> Yes, sure. You can do it after we push the frontend base code to the repo.

okay, Sir will we be notified of the same

**Comment by vaishnavig20** [comment]
Hey. I am a GSSOC'21 participant and can work on the UI as well as the frontend part.Would love to contribute to any feature of the landing page. Please assign it to me @Ayush-projects

**Comment by Ayush-projects** [comment]
Frontend folder has been added, create an issue for a particular change and go on :)
